apps/api-extractor/src/generators/DeclarationReferenceGenerator.ts

```diff
@@ -88,18 +89,9 @@ export class DeclarationReferenceGenerator {
   }
 
   private _getNavigationToSymbol(symbol: ts.Symbol): Navigation {
-    // resolve alias first:
-    if (symbol.flags & ts.SymbolFlags.Alias) {
-      symbol = this._collector.typeChecker.getAliasedSymbol(symbol);
-    }
-    // namespace always uses ".":
-    if (symbol.flags & ts.SymbolFlags.Namespace) {
-      return Navigation.Exports;
-    }
```

Comment:
I couldn't figure out why this was needed, so I removed it. It's also not always true that namespace always uses ".", you can have non-exported symbols within namespaces.

Reply:
I recall that in some situations, namespaces were not detected, because the method was called with an Alias, which is of course not a Namespace. But maybe some other change led to a behavior where this does not happen in the first place. Since the build-test scenarios all still work (I checked that ;-)), it seems fine to remove the code.